test/test_sparse_semi_structured.py

```diff
@@ -146,28 +146,46 @@ def forward(self, x):
                 x = x.contiguous()
                 return torch.nn.functional.relu(x)
 
-        def _test_mlp_contiguous_relu_compile(backend, dense_input_shape):
-            SparseSemiStructuredTensor._FORCE_CUTLASS = backend == "cutlass"
-            input = torch.rand(dense_input_shape, device="cuda").half()
+        SparseSemiStructuredTensor._FORCE_CUTLASS = backend == "cutlass"
```

Comment:
Shouldn't this be done in a context manager sort of thing?
This will modify this value for all subsequent tests

Reply:
Yes, each test currently sets this though so it won't cause an error. I plan to do some refactoring work splitting out the cuSPASRELt and CUTLASS subclasses, will address the flags in the tests then.